services/src/test/scala/cromwell/pipeline/service/impls/ProjectFileServiceTestImpl.scala

```diff
@@ -46,17 +54,23 @@ class ProjectFileServiceTestImpl(projectFiles: Seq[ProjectFile], testMode: TestM
   ): Future[List[ProjectFile]] =
     testMode match {
       case WithException(exc) => Future.failed(exc)
-      case _                  => Future.successful(projectFiles.toList)
+      case _ =>
+        val filesOpt = projectFileBundles.collectFirst {
+          case (id, files) if id == projectId => files
+        }
+        filesOpt match {
+          case Some(files) => Future.successful(files.toList)
+          case None        => Future.failed(new RuntimeException("No correct test file bundle received"))
```

Comment:
Why not just return an empty list? It sounds reasonable

Reply:
Fixed